js/js-overrides.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  // no full-page for target window by default
+  'check window with default fully with vg': {config: {branchName: 'js-sdk'}},
+  'check window with default fully with vg classic': {config: {branchName: 'js-sdk'}},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
  "check window with default fully with vg classic": {config: {branchName: 'js-sdk'}},
```